dir.traversal.targets

```diff
@@ -35,28 +35,20 @@
       <DefaultBuildAllTarget Condition="'$(DefaultBuildAllTarget)'==''">$(MSBuildProjectDefaultTargets)</DefaultBuildAllTarget>
     </PropertyGroup>
 
-   <!-- Clear ProjectWithConfiguration to prevent circular dependency -->
-   <ItemGroup><ProjectWithConfiguration Remove="@(ProjectWithConfiguration)" /></ItemGroup>
-   <MSBuild Targets="AnnotateProjectsWithConfiguration" 
-            Projects="@(Project)"> 
-     <Output TaskParameter="TargetOutputs" 
-             ItemName="ProjectWithConfiguration" /> 
-   </MSBuild> 
-
     <!-- To Serialize we use msbuild's batching functionality '%' to force it to batch all similar projects with the same identity
          however since the project names are unique it will essentially force each to run in its own batch -->
     <MSBuild Targets="$(DefaultBuildAllTarget)"
-             Projects="@(ProjectWithConfiguration)"
+             Projects="@(Project)"
              Condition="'$(SerializeProjects)'=='true' AND '%(Identity)' != ''"
              Properties="DefaultBuildAllTarget=$(DefaultBuildAllTarget);BuildAllProjects=true;BuildConfiguration=$(BuildConfiguration)"
-             ContinueOnError="ErrorAndContinue" />
+             ContinueOnError="ErrorAndStop" />
```

Comment:
Is this temporary? With it set like this it means that we will stop at the first failed project and not build the others.